src/main/java/uk/gov/hmcts/darts/audio/service/impl/AudioTransformationServiceImpl.java

```diff
@@ -248,16 +259,23 @@ private void processAudioRequest(Integer requestId) {
     }
 
     List<MediaEntity> filterMediaByMediaRequestTimeframe(List<MediaEntity> mediaEntitiesForRequest, MediaRequestEntity mediaRequestEntity) {
-        return mediaEntitiesForRequest.stream().filter(
-                media -> (mediaRequestEntity.getStartTime()).isBefore(media.getEnd())
-                    && (media.getStart().isBefore(mediaRequestEntity.getEndTime())))
+        Comparator<MediaEntity> mediaStartTimeChannelComparator = (media1, media2) -> {
```

Comment:
Just wondered if this is the right place to define this Comparator as it appears a bit confusing (to me at least!) having it here.

Reply:
I thought it was ok to keep close to the sorted stream operation, but I can push it out to a static field if it helps a bit.
We needed to do this as I noticed that the audio sessions were out of order in my download zip file.

Reply:
Done in commit b9a26437